documentation/RLC-quickstart.md

```diff
@@ -1,68 +1,75 @@
-Getting Started - Generate the RLC rest-level client libraries
-================================================================
+Getting Started - Generate the RLC rest-level client libraries with Cadl
+========================================================================
 
-# Prerequisites
+# Before you start
 
-You may refer to this [link](https://github.com/Azure/azure-sdk-for-js/blob/main/CONTRIBUTING.md#prerequisites) for the environment set up prerequisites in azure-sdk-for-js repository.
+Please refer to this [link](https://github.com/Azure/azure-sdk-for-js/blob/main/CONTRIBUTING.md#prerequisites) for the environment set up prerequisites in azure-sdk-for-js repository. We highly recommand to read [this blog](https://devblogs.microsoft.com/azure-sdk/azure-rest-libraries-for-javascript/) to get familiar with REST libraries for JavaScript. 
 
-# Project folder and name convention
+:warning: Note: if you’re still generating from Swagger with RLC, please read [this doc](./RLC-Swagger-quickstart.md) for Swagger specific details.
 
-Before we start, we probably should get to know the project folder and name convention for RLC libraries.
 
-1. Project Folder structure.  
-   normally, the folder structure would be something like `sdk/{servicename}/{servicename}-{modulename}-rest`. For example, we have `sdk/agrifood/agrifood-farming-rest` folder for Farmbeats account modules. That folder will be your **${PROJECT_ROOT} folder**.  
+# Project folder structure and name convention
```

Comment:
Maybe add a word to remind the customer to skip this part if they already worked on RLC before. Also this is a common knowledge not sure if we should repeat it in both docs.